features/verifysession/impl/src/main/kotlin/io/element/android/features/verifysession/impl/VerifySelfSessionStateMachine.kt

```diff
@@ -216,5 +219,8 @@ class VerifySelfSessionStateMachine @Inject constructor(
 
         /** Reset the verification flow to the initial state. */
         data object Reset : Event
+
+        /** Skip the verification flow. */
+        data object Skip : Event
```

Comment:
I would not touch the Verification state machine, skipping the verification should only affect the FTUE state. Except if we want to add a specific "Skipped verification" screen?